src/other/PropertyEditor.js

```diff
@@ -26,7 +26,8 @@
     PropertyEditor.prototype.publish("sectionTitle", "", "string", "Section Title",null,{tags:["Private"]});
     PropertyEditor.prototype.publish("defaultCollapsed", false, "boolean", "Default Collapsed Sections",null,{tags:["Private"]});
     PropertyEditor.prototype.publish("collapsibleSections", true, "boolean", "Collapsible Sections",null,{tags:["Private"]});
-    PropertyEditor.prototype.publish("excludeTags", ["Advanced","Private"], "array", "Array of publish parameter tags to exclude from PropertEditor",null,{tags:["Basic"]});
+    PropertyEditor.prototype.publish("excludeTags", ["Advanced","Private"], "array", "Array of publish parameter tags to exclude from PropertEditor",null,{tags:["Private"]});
+    PropertyEditor.prototype.publish("showProperties", ["fillOpacity","lineOpacity"], "array", "Array of publish parameter IDs to include in PropertEditor (all others will be excluded)",null,{tags:["Private"]});
```

Comment:
Yes, they all show if its empty, that wasn't intentionally committed. I'll remove it.

Though, there's something in the Widget.js constructor that sets array and widgetArray to **[]**